src/NUnitFramework/framework/Attributes/TestCaseAttribute.cs

```diff
@@ -384,7 +399,17 @@ private static void PerformSpecialConversions(object[] arglist, ParameterInfo[]
         /// <returns>One or more TestMethods</returns>
         public IEnumerable<TestMethod> BuildFrom(MethodInfo method, Test suite)
         {
-            return new TestMethod[] { new NUnitTestCaseBuilder().BuildTestMethod(method, suite, GetParametersForTestCase(method)) };
+            TestMethod test = new NUnitTestCaseBuilder().BuildTestMethod(method, suite, GetParametersForTestCase(method));
+            
+#if !PORTABLE
+            if (test.RunState != RunState.NotRunnable && !_platformHelper.IsPlatformSupported(this))
```

Comment:
I guess so... I just took a look at it and if you have both a PlatformAttribute and an IgnoreAttribute, the resulting RunState is indeterminate.

FWIW, this is a carryover from NUnit 2.x, when all attributes were applied in a known order. Now that they apply themselves, each one has to allow for the fact that other attributes may be present.

I'd say fix this instance and we can do a more systematic review later, possibly under the aegis of #374.